airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -105,6 +114,152 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
 
   const columnHelper = createColumnHelper<FullTableProps>();
 
+  const billingInsightsColumns = React.useMemo(() => {
+    return !isBillingInsightsEnabled
+      ? null
+      : [
+          columnHelper.accessor("connectionName", {
+            header: () => (
+              <SortableTableHeader
+                onClick={() => onSortClick("connectionName")}
+                isActive={sortBy === "connectionName"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.connection" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
+              <NavLink
```

Comment:
TODO: replace these with the new internal NavLink once merged.  I'm guessing they have hover styling baked in?  That was my main reason for not fiddling with that here right now!

Reply:
> I'm guessing they have hover styling baked in?

At the moment, no 🙁. Waiting for design feedback before adding anything. IMO we could go ahead and use the non-hover-styled `<Link>` now (it's been merged) and once styling is added it will just work™.

Any reason you're using `<NavLink>` here instead of just `<Link>`?

Reply:
Nope.  Will use Link.